arroyo-console/src/main.tsx

```diff
@@ -101,5 +103,10 @@ const theme = extendTheme(proTheme, {
   config: config,
 });
 
+if (process.env.NODE_ENV === 'development') {
```

Comment:
@mwylde this means that the mocks will apply with the service is run with `pnpm dev`. Is that ok?

Reply:
Is there a way to make that configurable? At least when I'm developing the frontend, I think I'll generally want to talk to a real arroyo api.

Reply:
I think when developing the frontend there are some real benefits to _not_ connecting it to a real arroyo api until the very end before merging. The main benefit with mocking the API is that you can deliberately develop loading and error states without needing to hack the api to somehow produce those states (which is especially hard if those states IRL are rare or short-lived, like loading).

But since `NODE_ENV` is just an environment variable you can always override it like this if you really want to:
```
NODE_ENV=production pnpm dev
```

and we also have `pnpm preview`.